src/test/integration/tests/Rounding.t.sol

```diff
@@ -0,0 +1,277 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.27;
+
+import "src/test/integration/IntegrationChecks.t.sol";
+
+contract Integration_Rounding is IntegrationCheckUtils {
+    using ArrayLib for *;
+    using StdStyle for *;
+
+    User attacker;
+    AVS badAVS;
+    IStrategy strategy;
+    IERC20Metadata token;
+    User goodStaker;
+    uint64 initTokenBalance;
+
+    OperatorSet mOpSet; // "manipOpSet" used for magnitude manipulation
+    OperatorSet rOpSet; // Redistributable opset used to exploit precision loss and trigger redistribution
+    
+    function _init() internal override {
+        _configAssetTypes(HOLDS_LST);
+
+        attacker = new User("Attacker"); // Attacker serves as both operator and staker
+        badAVS = new AVS("BadAVS"); // AVS is also attacker-controlled
+        strategy = lstStrats[0];
+        token = IERC20Metadata(address(strategy.underlyingToken()));
+        
+        // Prepares to add non-attacker stake into the protocol. Can be any amount > 0.
+        // Note that the honest stake does not need to be allocated anywhere, so long as it's in the same strategy.
+        goodStaker = new User("GoodStaker");
+        deal(address(token), address(goodStaker), uint256(1e18)); 
+        
+        // Register attacker as operator and create attacker-controlled AVS/OpSets
+        attacker.registerAsOperator(0);
+        rollForward({blocks: ALLOCATION_CONFIGURATION_DELAY + 1});
+        badAVS.updateAVSMetadataURI("https://example.com");
+        mOpSet = badAVS.createOperatorSet(strategy.toArray()); // setup low mag operator
+        rOpSet = badAVS.createRedistributingOperatorSet(strategy.toArray(), address(attacker)); // execute exploit
+
+        // Register for both opsets
+        attacker.registerForOperatorSet(mOpSet);
+        attacker.registerForOperatorSet(rOpSet);
+        
+        _print("setup");
+    }
+
+    // TODO: consider incremental manual fuzzing from 1 up to WAD - 1
+    function test_rounding_allMagsSlashed(uint16 slashes, uint64 initialWadToSlash, uint64 _initTokenBalance, uint24 r) public rand(r) {
```

Comment:
IMO we should choose a lane for param generation:

1. either use `rand(r)` and the `_randUint` method to generate _all_ parameters
2. or allow forge to generate all parameters and if we need additional randomness, use `cheats.randomUint` and similar

#1 means we should do sequential runs, supplying (1, 2, 3, 4, etc)
#2 means we just let forge decide and we only control the number of runs

Reply:
Or I guess there's a way to do both - let forge generate parameters via #2, but run tests using `forge t --fuzz-seed <1, 2, 3, ...>`

That may be most convenient?